tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -195,77 +188,69 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         };
     };
 
-    fluid.tests.assertVisibleTips = function (pager, sequence, message, targetIds, expectedTooltip) {
-        sequence.push({
-            event: "{trackTooltips}.events.notifyFocusChange",
-            listener: function () {
-                fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
-                    jqUnit.assertNode("When the page size is " + pager.model.pageSize + ", The contents of the tooltip should be set", expectedTooltip,
-                        $("b", tooltip));
-                });
-            }
-        });
+    fluid.tests.getTooltipContents = function (listOfData, pageSize) {
+        var tooltipContents = [];
+        for (var i = 0; i < listOfData.length; i += pageSize) {
+            var tooltipContent = [];
+            var pageStartIndex = i;
+            var pageEndIndex = Math.min(pageStartIndex + pageSize - 1, listOfData.length - 1);
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageStartIndex].text});
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageEndIndex].text});
+            tooltipContents.push(tooltipContent);
+        }
+
+        return tooltipContents;
     };
 
-    /**
-     * Change the page size
-     */
-    fluid.tests.setPageSize = function (pager, sequence, pageSize) {
-        sequence.push({
-            func: function () {
-                pager.locate("pageSize").val(pageSize);
-            }
-        });
-        sequence.push({
-            element: "#rendered-ioc .flc-pager-page-size",
-            jQueryTrigger: "change"
-        });
+    fluid.tests.verifyLinksAndTooltips = function (pager, pageSize, linksList) {
+        fluid.tests.verifyPagerBar(pager, "top", pageSize);
+        fluid.tests.verifyPagerBar(pager, "bottom", pageSize);
+        fluid.tests.verifyToolTipsOfPagerBar(pager, "top", pageSize,
+            fluid.tests.getTooltipContents(linksList, pageSize));
+        fluid.tests.verifyToolTipsOfPagerBar(pager, "bottom", pageSize,
+            fluid.tests.getTooltipContents(linksList, pageSize));
```

Comment:
fluid.tests.getTooltipContents seems to be called twice, you should just store the result as a variable and use that as an argument to fluid.tests.verifyToolTipsOfPagerBar